### ISSUE TITLE
New blog post does not show up in fleek build preview if commit was amended

### ISSUE THREAD

**Issue by meandavejustice** [ipfs/ipfs-blog#526]
It appears that blog posts are not built properly in ci if the commit was amended. This is currently happening with https://github.com/ipfs/ipfs-blog/pull/510 and happened previously with https://github.com/ipfs/ipfs-blog/pull/509, to solve last time I opened a fresh duplicate pr, https://github.com/ipfs/ipfs-blog/pull/520. I don't want to keep cluttering up the pr queue with these duplicate prs 😬. 

This may be an upstream issue with Fleek's workflow actions https://github.com/FleekHQ/workflows, but logging here until I get more info

**Comment by vesahc** [comment]
Hi @meandavejustice, sorry I missed this. This has been an ongoing issue with many Fleek sites and I've been in contact with them to resolve it. We're making progress, but we still might see a few stuck PR's occasionally. 

As you've figured out the best way is to create a new PR. There is another workaround that can unblock stuck builds in cases where we need to keep comments, approvals, etc. 

> I found a workaround! You need to create a new PR for the same commit using different branch name. This will kick the build upon completion both original and new PR will get status update. Then you can merge the original and close the new one.
> 
> 
> If PR was created from remote repository you can pull the ref to your repository locally by following these instructions:
> 
> https://help.github.com/articles/checking-out-pull-requests-locally/
> 
> git fetch origin pull/{id}/head:temporary
> 
> git push origin temporary
>
source: https://www.notion.so/protocollabs/Fleek-Filecoin-Deploys-Broken-1041a2bb095b4c198155225860144112?pvs=4#49f80e57d96a4057be80793dbcfeebab
